UI/window-basic-main-outputs.cpp

```diff
@@ -349,6 +349,36 @@ void SimpleOutput::LoadStreamingPreset_Lossy(const char *encoderId)
 	obs_encoder_release(videoStreaming);
 }
 
+/* mistakes have been made to lead us to this. */
```

Comment:
```suggestion
/* Mistakes have been made to lead us to this. */
```

Reply:
It's not a big deal if some comments start with lowercase. They're just comments.